### ISSUE TITLE
Add TensorFlow Ops for T5

### ISSUE THREAD

**Issue by vlasenkoalexey** [NVIDIA/FasterTransformer#235]
### Description

```shell
I saw you recently added T5 kernels for PyTorch here: https://github.com/NVIDIA/FasterTransformer/tree/main/src/fastertransformer/th_op/t5
Would it be possible to also add corresponding Ops for TensorFlow?
```

### Reproduced Steps

```shell
See description.
```

**Comment by byshiue** [comment]
Thanks for your feedback, we will consider it.

**Comment by rossbucky** [comment]
@byshiue will the FT op be in the roadmap for the next release? TF op turns out to be faster than th op from the decoder(decoding) benchmark and is easier for production usage. Thanks.

**Comment by byshiue** [comment]
> @byshiue will the FT op be in the roadmap for the next release? TF op turns out to be faster than th op from the decoder(decoding) benchmark and is easier for production usage. Thanks.

Thank you, we will consider. 

But for benchmark on TF op and PyTorch op, their performance are similar in FT. For some cases, FT TF op is faster, for some cases, FT PyTorch op is faster. 

And for production usage, we recommend trying the triton backend, which helps batch the request and support several frameworks as backend.

**Comment by rossbucky** [comment]
> > @byshiue will the FT op be in the roadmap for the next release? TF op turns out to be faster than th op from the decoder(decoding) benchmark and is easier for production usage. Thanks.
> 
> Thank you, we will consider.
> 
> But for benchmark on TF op and PyTorch op, their performance are similar in FT. For some cases, FT TF op is faster, for some cases, FT PyTorch op is faster.
> 
> And for production usage, we recommend trying the triton backend, which helps batch the request and support several frameworks as backend.

Yes. The FT native triton backend is very cool if the usage is only the generation. one advantage of using TF op is that we can combine it with other TF ops and export it as a whole TF model. We found it very useful since adding this custom TF op in triton tf backend or tfserving is also straightforward and easy

**Comment by byshiue** [comment]
This feature is supported in latest v5.2 release.
